src/main/java/org/mastodon/mamut/feature/branch/BranchDisplacementDurationFeature.java

```diff
@@ -47,6 +47,110 @@
+ * the branchspot and the last timepoint of the previous branchspot in the
+ * lineage tree.
+ * <p>
+ * For roots, it is computed as the difference in timepoints between the last
+ * timepoint of the branchspot and the first timepoint the previous branchspot
+ * in the lineage tree.
+ * <p>
+ * Cf. following example:
+ *
+ * <h1>Spot-Graph</h1>
+ * 
+ * <pre>
+ *                                                Spot( 0, X=1,00, Y=2,00, Z=3,00, tp=0 )
+ *                                                                   │
+ *                                                                   │
+ *                                                Spot( 1, X=2,00, Y=4,00, Z=6,00, tp=1 )
+ *                                                                   │
+ *                                                                   │
+ *                                                Spot( 2, X=3,00, Y=6,00, Z=9,00, tp=2 )
+ *                       ┌───────────────────────────────────────────┴──────────────────────┐
+ *                       │                                                                  │
+ *  Spot( 11, X=12,00, Y=24,00, Z=36,00, tp=3 )                         Spot( 3, X=4,00, Y=8,00, Z=12,00, tp=3 )
+ *                       │                                                                  │
+ *                       │                                                                  │
+ *  Spot( 12, X=13,00, Y=26,00, Z=39,00, tp=4 )                         Spot( 4, X=5,00, Y=10,00, Z=15,00, tp=4 )
+ *                       │                                            ┌─────────────────────┴─────────────────────┐
+ *                       │                                            │                                           │
+ *  Spot( 13, X=14,00, Y=28,00, Z=42,00, tp=5 )   Spot( 8, X=9,00, Y=18,00, Z=27,00, tp=5 )   Spot( 5, X=6,00, Y=12,00, Z=18,00, tp=5 )
+ *                                                                    │                                           │
+ *                                                                    │                                           │
+ *                                               Spot( 9, X=10,00, Y=20,00, Z=30,00, tp=6 )   Spot( 6, X=7,00, Y=14,00, Z=21,00, tp=6 )
+ *                                                                    │                                           │
+ *                                                                    │                                           │
+ *                                               Spot( 10, X=11,00, Y=22,00, Z=33,00, tp=7 )  Spot( 7, X=8,00, Y=16,00, Z=24,00, tp=7 )
+ * </pre>
+ * 
+ * <h1>BranchSpot-Graph</h1>
+ * 
+ * <pre>
+ *                        branchSpot0
+ * 	       ┌──────────────┴─────────────────┐
+ * 	       │                                │
+ * 	   branchspot1                      branchSpot2
+ * 	┌──────┴───────┐
+ * 	│              │
+ * branchspot3 branchSpot4
```

Comment:
Why is the BranchGraph flipped, when compared to the ModelGraph?

Reply:
This was a mistake. Changed it.